### ISSUE TITLE
Backup error with multiple sources defined in the config file

### ISSUE THREAD

**Issue by nickchomey** [rustic-rs/rustic#475]
I'm getting this error now whenever I run a backup that has multiple sources defined in the config.

```
nick@DESKTOP-DIFRTR1:~$ rustic -r rclone:onedrive:wsl2 backup --dry-run
using config /home/nick/.config/rustic/rustic.toml
[00:00:01] reading index...               ████████████████████████████████████████         18/18
[00:00:01] getting latest snapshot...     ████████████████████████████████████████         14/14
[00:00:00] backing up...                  ████████████████████████████████████████         0B/0B     0B/s  ( ETA -)
Files:       1 new, 0 changed, 0 unchanged
Dirs:        3 new, 0 changed, 0 unchanged
Added to the repo: 460 B (raw: 402 B)
processed 1 files, 0 B
snapshot 00000000 successfully saved.
thread 'main' panicked at 'removal index (is 1) should be < len (is 1)', src/commands/backup.rs:183:36
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Here's the config that generates the error

```
[global]
log-level = "off" # log levels: trace, debug, info, warn, error or off
#log-file = "/log/rustic.log"

# repository options: These options define which backend to use and which password to use.
[repository]
repository = "rclone:onedrive:wsl2"
password = ""

[[backup.sources]]
source = "/home/nick/test"

[[backup.sources]]
source = "/home/nick/test2"
```

I'm also noticing that depsite doing a backup, it only processes the first source (which has 1 file) and ignores/fails on the second source (which has 2 files)

If I comment out either of the sources like this, it will work.
```
[[backup.sources]]
source = "/home/nick/test"

#[[backup.sources]]
#source = "/home/nick/test2"
```

I'm using `rustic v0.4.4-25-g8870bd6`

**Comment by aawsome** [comment]
Thanks for reporting - this is actually a bug introduced by #458 and the fact that all paths from the config are now sanitized. Should be fixed by #476 in few minutes!

**Comment by aawsome** [comment]
Yes, this is how it should behave! Note that it will create two snapshots.
